### PR TITLE
[FIX] mtl not converging for smaller alphas on path

### PR DESCRIPTION
**Pull request by mathurinm**

### CODE REVIEW

File: celer/cython_utils.pyx [hunk 1]
```diff
@@ -226,6 +226,7 @@ cdef int create_accel_pt(
     cdef int info_dposv
 
     cdef int i, j, k
+    # warning: this is wrong (n_samples) for MTL, is is handled outside
```
Comment (agramfort) [inline celer/cython_utils.pyx:229]:
```suggestion
    # warning: this is wrong (n_samples) for MTL, it is handled outside
```

**Comment by agramfort** [comment]
I think you should also add a test

**Comment by mathurinm** [comment]
@agramfort yes, and I need to do the maths properly to check the scaling (also in Lasso and Logreg). One current test is slow and I need to run it in verbose mode to check.

But I hope this will already be enough for @ja-che experiments

**Comment by agramfort** [comment]
Ok

@ja-che can you try ?

**Comment by ja-che** [comment]
Thanks @mathurinm, it seems much better now ! Close to 10x faster on my problems.

**Comment by mathurinm** [comment]
Merci @ja-che

**Comment by agramfort** [comment]
thanks @mathurinm !

**Comment by ja-che** [comment]
Thanks @mathurinm

**Pull request merged**